src/helpers/manage-merge-queue.ts

```diff
@@ -63,7 +63,7 @@ export const manageMergeQueue = async ({
     return removePrFromQueue(pullRequest);
   }
   const queuedPrs = await getQueuedPullRequests();
-  const queuePosition = queuedPrs.length;
+  const queuePosition = queuedPrs.length + 1;
```

Comment:
Length isn't an offset. Why the `+ 1`?

Reply:
`getQueuedPullRequests()` will now return a list of open PRs with both the `READY FOR MERGE` and `QUEUED FOR MERGE #X` labels. Before, this was _including_ the PR we are running this for

Reply:
So now, if there are 2 PRs in the queue, and this runs for a new PR that just got `READY FOR MERGE`, `queuedPrs.length` should be 2, and thus `queuePosition` for this new PR should be 3.